pages/common/cargo-doc.md

```diff
@@ -0,0 +1,24 @@
+# cargo doc
+
+> Build and view Rust package documentation offline.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-doc.html>.
+
+- Build and view the current packages documentation in your browser:
+
+`cargo doc --open`
+
+- View a particular packages documentation:
+
+`cargo doc --open --package {{crate_name}}`
+
+- View all workspace package documentation:
+
+`cargo doc --open --workspace`
+
+- Build documentation without accessing the network:
+
+`cargo doc --offline`
+
+- View a particular dependencies documentation offline:
```

Comment:
Unclear like the ones above, please rephrase. Do you meant documentation of a particular dependency, or something else?